frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx

```diff
@@ -7,6 +7,57 @@ import { MockedAsset } from "@/store/assets/assets.types";
 
 const NoSSRChart = dynamic(() => import("react-apexcharts"), { ssr: false });
 
+type PriceChartLabelProps = {
+  priceSeries: [number, number][];
+  predictedPriceSeries: [number, number][];
+  assetSymbol?: string;
+};
+
+const PriceChartLabels = ({
+  priceSeries,
+  predictedPriceSeries,
+  assetSymbol,
+}: PriceChartLabelProps) => {
+  if (priceSeries.length > 0 && predictedPriceSeries.length > 0) {
+    return (
+      <>
+        <FiberManualRecordIcon color="primary" />
+        <Typography variant="body2" pl={1} pr={2}>
+          {assetSymbol}
+        </Typography>
+        <FiberManualRecordIcon color="inherit" />
+        <Typography variant="body2" pl={1} whiteSpace="nowrap">
+          {assetSymbol} predicted price (without new buyers)
+        </Typography>
+      </>
+    );
+  }
+
+  if (priceSeries.length === 0 || predictedPriceSeries.length === 0) {
+    if (priceSeries.length === 0) {
+      return (
+        <>
+          <FiberManualRecordIcon color="primary" />
+          <Typography variant="body2" pl={1} pr={2}>
+            {assetSymbol} predicted price (without new buyers)
+          </Typography>
+        </>
+      );
+    }
+    if (predictedPriceSeries.length === 0) {
+      return (
+        <>
+          <FiberManualRecordIcon color="primary" />
+          <Typography variant="body2" pl={1} pr={2}>
+            {assetSymbol}
+          </Typography>
+        </>
+      );
+    }
+  }
+  return null;
+};
```

Comment:
```suggestion
  if (priceSeries.length > 0 && predictedPriceSeries.length > 0) {
    return null;
  }
  
  return (
    <>
      <FiberManualRecordIcon color="primary" />
      <Typography variant="body2" pl={1} pr={2}>
        `${assetSymbol} ${priceSeries.length === 0 ? "predicted price (without new buyers)" : ""}`
      </Typography>
    </>
  )
```

Reply:
@andresmechali `priceSeries` and `predictedPriceSeries` we have to show labels for both series, similarly, if data exists within one of the two we want to show only that label